### PR TITLE
Clarify policies in README.md

### PR DESCRIPTION
**Pull request by jackieokay**
I'm updating this in README.md on master because that's what most people see when they land on this repository.

@tfoote does this look consistent with what we discussed?

### CODE REVIEW

**Comment by dirk-thomas** [comment]
This seems to address #23.

File: README.md [hunk 1]
```diff
@@ -12,3 +12,31 @@ The two branches should be maintained as closely as possible.
...
+
+##Guide to blacklisting packages
+
+If you wish to blacklist a package on build.ros.org, submit a pull request to `production` doing the following:
+
+Find the `release-build.yaml` file which is appropriate for the platform you wish to blacklist your package for under the folder for the appropriate distribution.
+
+For example, if you want to blacklist a package on 32-bit ARM in Kinetic Kame, you want to edit the file `kinetic/release-armhf.build`.
+
+Add the name of the package to blacklist under the entry `package_blacklist`, e.g.
```
Comment (dirk-thomas) [inline README.md]: Maybe mention alphabetical order.

File: README.md [hunk 2]
```diff
@@ -12,3 +12,31 @@ The two branches should be maintained as closely as possible.
...
+
+```
+  package_blacklist:
+    - my_blacklisted_package
+```
+
+When you submit a pull request, please *open a ticket* in `ros_buildfarm_config` or the repository for your package describing why you are blacklisting your package, and link to that ticket in your pull request to blacklist the package.
```
Comment (dirk-thomas) [inline README.md]: The requested information could just be provided in the pull request description. Why does a user need to create two things, a ticket _and_ a pull request?

**Comment by tfoote** [comment]
updated based on dirk's feedback

**Pull request merged**